rpc/src/module/test.rs

```diff
@@ -512,10 +513,9 @@ impl IntegrationTestRpc for IntegrationTestRpcImpl {
     fn process_block_without_verify(&self, data: Block, broadcast: bool) -> Result<Option<H256>> {
         let block: packed::Block = data.into();
         let block: Arc<BlockView> = Arc::new(block.into_view());
-        let ret = self
+        let ret: VerifyResult = self
```

Comment:
no need to add type annotation and we can remove `use ckb_chain::VerifyResult;`.